integration_test.go

```diff
@@ -127,12 +130,12 @@ func testOrbslamInternalState(t *testing.T, svc slam.Service, dataDir string) {
 
 	// Save the data from the call to GetInternalStateStream for use in next test.
 	timeStamp := time.Now()
-	filename := filepath.Join(dataDir, "map", "orbslam_int_color_camera_data_"+timeStamp.UTC().Format(slamTimeFormat)+".osa")
-	err = os.WriteFile(filename, internalStateStream, 0644)
+	filename := filepath.Join(dataDir, "map", "orbslam_int_color_camera_data_"+timeStamp.UTC().Format(dataprocess.SlamTimeFormat)+".osa")
+	err = os.WriteFile(filename, internalStateStream, 0o644)
```

Comment:
Nope -- both of these are notations for octal (https://go.dev/ref/spec#Integer_literals). However, the 0o6400 notation is the suggested way of writing this now because it is more explicit (the gofmt linter will also automatically make this change)

Reply:
Ah, thanks!